back/engines/commercial/bulk_import_ideas/app/jobs/bulk_import_ideas/idea_import_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module BulkImportIdeas
+  class IdeaImportJob < ApplicationJob
+    self.priority = 60
+
+    CONSTANTIZER = {
+      'xlsx' => {
+        exporter_class: Exporters::IdeaXlsxFormExporter
+      },
+      'pdf' => {
+        exporter_class: Exporters::IdeaPdfFormExporter
+      }
+    }
```

Comment:
```suggestion
    EXPORTERS = {
      'xlsx' => Exporters::IdeaXlsxFormExporter,
      'pdf' => Exporters::IdeaPdfFormExporter
    }
```
Do we expect any more classes here? If not, it could be a bit simpler.